x/bridge/types/params.go

```diff
@@ -0,0 +1,117 @@
+package types
+
+import (
+	"fmt"
+
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+
+	"github.com/osmosis-labs/osmosis/osmoutils"
+)
+
+var (
+	KeySigners = []byte("Signers")
+	KeyAssets  = []byte("Assets")
+)
+
+func NewParams(signers []string, assets []AssetWithStatus) Params {
+	return Params{
+		Signers: signers,
+		Assets:  assets,
+	}
+}
+
+// DefaultParams creates default x/bridge params.
+func DefaultParams() Params {
+	return Params{
+		Signers: []string{}, // TODO: what to use as default?
```

Comment:
Blank signers as default makes sense to me.

Or could even default to the gov address as the sole signer.